### DISCUSSION TITLE
videos/mspider/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1197] [Announcements]
# videos/mspider/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/videos/mspider/

**Comment by liyuerich** [comment]
点进来后没有注意到右侧有导航，还以为只有1个视频。 能否标题下也有个content index？

观看视频的过程中如果有滑动，会有其他视频也播放， 也不知道是播放的哪一个，只好一个一个的找，然后暂停

**Reply by samzong** [reply]
@Michelle951 可以优化下内容。

**Reply by yulng** [reply]
可能需要讲一下道理，而不是操作

**Comment by liyuerich** [comment]
安全治理,istio版本升级 配音能否改成系统自动的声音，和其他视频一致。 自己录制的没有办法那么完美，有杂音

**Comment by liyuerich** [comment]
视频点击到某操作步骤的时候可以说明一下这个参数的大概概念，给更多信息。 安全治理 这个视频这块做的比较好

**Comment by liyuerich** [comment]
录制的时候能否在一个简单的环境，因为过程中还看到企业微信的信息了。 :-(
